src/colvarscript.cpp

```diff
@@ -324,6 +324,22 @@ int colvarscript::proc_colvar(colvar *cv, int objc, unsigned char *const objv[])
     return COLVARS_OK;
   }
 
+  if (subcmd == "getatoms") {
```

Comment:
I am using the old-style scripting interface here, mostly because of the list-of-lists structure of the data, for which I have no good generic solution. I'm happy to reimplement this in the generic form when we have a converged way to handle the data structures.

Reply:
That's reasonable for now.

Reply:
P.S. Can you change to `getatomids`?